### ISSUE TITLE
How to use this?

### ISSUE THREAD

**Issue by reagle** [guillermooo/sublime-troubleshooting#3]
There's no documentation on how to use.

**Comment by FichteFoll** [comment]
This is true, since the package is still kind of unfinished and many things still need to be implemented.

Currently, the only thing this package does is prepare a bug report via the "Troubleshooting: Generate Bug Report Template" command from the command palette.

**Comment by guillermooo** [comment]
@reagle It also lets you conveniently turn on and off different logging options for debugging.
